spec/index.bs

```diff
@@ -3189,10 +3210,13 @@ The {{GPUMapMode}} flags determine how a {{GPUBuffer}} is mapped when calling
                     Issue(gpuweb/gpuweb#3271): This describes one possible behavior.
                     Need to discuss whether it's the right one.
                 1. [=Assert=] |deviceTimelineStateAtCompletion| is "[=buffer device timeline state/unavailable=]".
-                1. Let |m| be a new {{ArrayBuffer}} of size |rangeSize|.
-                1. Set the content of |m| to |dataForMappedRegion|.
+                1. Let |data| be [$CreateByteDataBlock$](|rangeSize|).
+                1. If the allocations of |data| or |dataForMappedRegion| failed:
```

Comment:
Not clear to me if `|dataForMappedRegion|` is considered an "allocation" here? Given that we're already validating the validity of the buffer above is there ever a point where `|dataForMappedRegion|` could fail to be produced?

Reply:
I don't know, I think we can just pretend it's infallible in the spec. Removed.